src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -52,5 +52,7 @@ internal class LocalizableStrings
                                         Logs are written to the provided file.";
 
         public const string CmdNoBuildDescription = @"Do not build project before testing.";
+
+        public const string RunSettingsArgsHelpText = @"Any extra options that should be passed to MSBuild or adapter. See 'dotnet msbuild -h' and 'dotnet vstest --help' for available options.";
```

Comment:
The string doesn't seems correct. This argument is not for adapter but for commandline runsttings argument of vstest.
It should be some thing like "Any extra options that will be consider as commndline runsettings argument or will pass to msbuild. See 'dotnet msbuild -h' and 'dotnet vstest --help' for available options.
Example: dotnet test TestProject.csproj -- RunConfiguration.ResultsDirectory=ResultsDirectoryName"